src/Components/ClusterHeader/ClusterHeader.spec.ct.js

```diff
@@ -43,6 +44,7 @@ describe('cluster page header', () => {
     // check last seen text
     cy.get(LAST_SEEN_FIELD).should('have.text', '24 Jul 2021 14:22 UTC');
   });
+  //FIXME spinner does not affect UUID/last seen?
```

Comment:
uuid is rendered immediately (know from URL). however, last seen and cluster display name are taken from API response. if API request is ongoing, the skeleton rendered for both fields. if suceeds, names and timestamp written. if last seen unavailable, then Unknown rendered. if cluster name empty/unavailable, the cluster id is rendered in the title

Reply:
Nice trick. Thanks for explaining